modules/ROOT/pages/indexes/search-performance-indexes/overview.adoc

```diff
@@ -0,0 +1,24 @@
+:description: Overview of the search-performance indexes available in Neo4j.
+= Search-performance indexes
+
+Search-performance indexes enable quicker retrieval of exact matches between an index and the primary data storage.
+There are four different search-performance indexes available in Neo4j:
+
+* *Range indexes*: Neo4j’s default index.
+Supports most types of predicates, including node label and relationship types.
+
+* *Text indexes*: solves predicates operating on `STRING` values.
+Optimized for queries filtering `STRING` properties for what they `CONTAIN` or `ENDS WITH`.
+
+* *Point indexes*: solves predicates on spatial `POINT` values.
+Optimized for queries filtered on distance or within bounding boxes.
+
+* *Token lookup indexes*: only solves node label and relationship type predicates (i.e. they cannot solve any predicates filtered on properties).
+Two token lookup indexes (one for node labels and one for relationship types) are present when a database is created in Neo4j.
+
+To learn more about creating, listing, and deleting these indexes, as well as more details about the predicates supported by each index type, see xref:indexes/search-performance-indexes/managing-indexes.adoc[Managing search-performance Indexes].
```

Comment:
If you don't want to give a special label to the link, it's best to put the xref with empty `[]`. In this way, if the title of the target xref is changed, the link label changes automatically.